### PR TITLE
Updated the settings views to use Swift and storyboards

### PR DESCRIPTION
**Pull request by jamie-conlon**
Storyboards
- The initial storyboard is the main settings view. This is a static tableview.
- The second storyboard is the sync settings view. This is a dynamic tableview where we can have various syncing services with different settings associated to each one.

Swift
- Code is pretty much the same as obj-c but just change to Swift syntax.

### CODE REVIEW

File: Classes/Settings/SettingsController.swift [hunk 1]
```diff
@@ -0,0 +1,113 @@
+//
+//  TestSettingsController.swift
+//  MobileOrg
+//
+//  Created by Jamie Conlon on 04/01/2017.
+//  Copyright © 2017 Sean Escriva. All rights reserved.
```
Comment (mgmart) [inline Classes/Settings/SettingsController.swift]: Should be © 2017 Jamie Conlon, shouldn't it?

File: Classes/Settings/SettingsController.swift [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+//
+
```
Comment (mgmart) [inline Classes/Settings/SettingsController.swift]: GNU License Header missing

File: Classes/Settings/SettingsController.swift [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+import Foundation
+import UIKit
+
+class SettingsController: UITableViewController {
+    
+    @IBOutlet weak var appBadgeSwitch: UISwitch!
```
Comment (mgmart) [inline Classes/Settings/SettingsController.swift]: The existing Swift files have a 2 spaces indentation. Would be nice if we could keep this consistent.

**Comment by webframp** [comment]
I've used this in the past to define project editor settings: http://editorconfig.org/

Requires a plugin with Xcode though

**Comment by mgmart** [comment]
Yeah, and to use plugins you have to unsign Xcode nowadays :(

**Comment by webframp** [comment]
You may want this: https://help.github.com/articles/remove-sensitive-data/ 

We can't delete the PR, but if you rewrite what's in the branch this PR is based on you'll be fine

**Comment by jamie-conlon** [comment]
I deleted the whole branch and created a new branch with the same name (I know this was pretty stupid but I was very confused at this stage!) So I don't think I'll be able to rewrite over this PR... 
I've changed my dropbox api keys so thats not really an issue. 

My new PR should have no sensitive info in it. I think...
git 1 - Jamie 0...

**Pull request closed without merging**